frontend/src/molecules/FormField.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { useField } from 'formik';
+
+import { Field } from './';
+
+export function FormField({ name, ...props }) {
```

Comment:
Molecule `Field` is now used only here - I would merge this molecule `FormField` and `Field` into one molecule. It will also fix usage one molecule in other - I think it's now good for example project for students.

Reply:
Hmm I think it is better to have Field as a component view molecule, because if you want to use only the component without Formik you will not have that option. Maybe the naming is not good.

Reply:
I agree with two components, one pure React, second connecting to Formik.
Maybe this could be renamed to `FormikField`.